l10n_th_vat_report/data/report_data.xml

```diff
@@ -0,0 +1,26 @@
+<odoo>
+    <report string='VAT Report'
+            id='action_vat_report_html'
+            model='report.vat'
+            name='l10n_th_vat_report.report_vat_report_base'
+            file='l10n_th_vat_report.report_vat_report_base'
+            report_type='qweb-html'
+            attachment_use='False'/>
+
+    <report string='PDF'
+            id='action_vat_report_pdf'
+            model='report.vat'
+            name='l10n_th_vat_report.report_vat_report_base'
+            file='l10n_th_vat_report.report_vat_report_base'
+            report_type='qweb-pdf'
+            print_report_name='"VAT Report - [%s]" % object.date_range_id.display_name'
+            paperformat='l10n_th_vat_report.paperformat_vat_report'/>
+
+    <report string='XLSX'
+            id='action_vat_report_xlsx'
+            model='report.vat'
+            name='l10n_th_vat_report.report_vat_report_xlsx'
+            file='VAT Report'
+            report_type='xlsx'
```

Comment:
Can we have this too for XLSX?
            print_report_name='"VAT Report - [%s]" % object.date_range_id.display_name'

Reply:
print_report_name can't use for XLSX.

Reply:
I am proposing the fix here https://github.com/OCA/reporting-engine/pull/315
You can use it to test. I think you can just set it same way, when it is merged it will just work.